File: server/SprintBoard.Api/Models/Comment.cs
// Models/Comment.cs
namespace SprintBoard.Api.Models;

public class Comment
{
    public int Id { get; set; }

    public int IssueId { get; set; }
    public Issue Issue { get; set; } = null!;

    public int AuthorId { get; set; }
    public User Author { get; set; } = null!;

    public string Content { get; set; } = string.Empty;

    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    public DateTime? UpdatedAt { get; set; }
}

File: client/src/types/Comment.ts
import type { Assignee } from "./Issue";

export interface Comment {
  id: number;
  issueId: number;
  content: string;
  author: Assignee;
  createdAt: string;
  updatedAt: string | null;
}

export interface CreateCommentPayload {
  content: string;
}

export interface UpdateCommentPayload {
  content: string;
}
